components/net/http_loader.rs

```diff
@@ -731,7 +731,7 @@ async fn obtain_response(
                             (connect_end - connect_start).unsigned_abs(),
                             (send_end - send_start).unsigned_abs(),
                             is_xhr,
-                            browsing_context_id,
+                            browsing_context_id.unwrap(),
```

Comment:
We need another `if let` around prepare_devtools_request instead of using unwrap here.